Gemfile.lock

```diff
@@ -523,4 +523,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.1.4
+   2.3.22
```

Comment:
I'm wondering if this change is the problem that's causing the tests to fail.

Reply:
Yeah; I was going to give it a bit, but that seemed like the most likely.  Not sure why that would mean that `rake` was unavailable, though.

Reply:
Yup; it looks like it was unhappy about the new bundler version..... :shrug: